src/sentry/event_manager.py

```diff
@@ -97,9 +97,6 @@ def get_hashes_for_event_with_reason(event):
     # here because `EventManager.save` pops off the `message` attribute before
```

Comment:
Not sure this comment is still necessary since we removed the block below. I think the PR history provides enough context that this point. (Sorry this is directly contradictory to what I said before)